tests/js/server/aql/aql-filter-projections.js

```diff
@@ -73,7 +73,7 @@ function filterProjectionsPlansTestSuite () {
         assertEqual(1, nodes[0].indexes.length, query);
         assertEqual(query[1], nodes[0].indexes[0].type, query);
         assertEqual(normalize(query[2]), normalize(nodes[0].projections), query);
-        assertEqual(normalize([]), normalize(nodes[0].filterProjections), query);
+        assertEqual(normalize(query[3]), normalize(nodes[0].filterProjections), query);
```

Comment:
Some of the queries above don't have their member[3] set. I guess this is ok because we are then comparing against `normalize(undefined)`, which should produce `[]`. So it's ok, but I was slightly confused in the beginning when looking at the change.

Reply:
I double checked and I'm sure all of them have a member[3].